src/platform/silabs/wifi/SiWx/WifiInterfaceImpl.cpp

```diff
@@ -114,6 +114,32 @@ osMessageQueueId_t sWifiEventQueue = nullptr;
 sl_net_wifi_lwip_context_t wifi_client_context;
 sl_wifi_security_t security = SL_WIFI_SECURITY_UNKNOWN;
 
+// Convert sl_wifi_security_t to wfx_sec_t
```

Comment:
- this function looks like coming in between the variables. Move this after the variables are done.